### ISSUE TITLE
Report to identify released items that had embargo dates that violated the length restriction set by the collection

### ISSUE THREAD

**Issue by amyehodge** [sul-dlss/happy-heron#3625]
I'm looking for H2 items where 1) an embargo was set, 2) that embargo was past the allowed date as indicated by the current collection setting, and 3) the embargo has already expired. 

Note that we are going to assume the collection setting hasn't changed, since it probably hasn't, and not try to identify the collection setting at the time the embargo was set. Also, we think the problem we are anticipating with making further edits would only be a problem if the current collection setting is violated.

This is to make sure there aren't lingering problems as a result of the bug identified and recently fixed by https://github.com/sul-dlss/happy-heron/issues/3621.

@andrewjbtw FYI

**Comment by amyehodge** [comment]
From @mjgiarlo:
When I run the same report ignoring whether the embargo has already released, I get some offenders:
[[true, Sun, 07 Sep 2025, Tue, 07 Mar 2023, "druid:bg246gz1933"],
 [true, Sat, 15 Mar 2025, Sun, 09 Feb 2025, "druid:vj832cf1556"],
 [true, Fri, 12 Sep 2025, Sun, 12 Mar 2023, "druid:qx469wp3917"],
 [true, Wed, 31 Dec 2025, Tue, 18 Mar 2025, "druid:zb733jx3137"],
 [true, Thu, 03 Jun 2027, Sun, 23 Mar 2025, "druid:zx198bb7198"]]

**Comment by amyehodge** [comment]
My assessment of these, with comments on the first one as well from @andrewjbtw:
- druid:bg246gz1933: Rigler and Deutsch from Jerry Persons. Embargo was released manually via Argo in 2022. This should not be a problem in the future as we are not managing via H2 any more.
- druid:vj832cf1556: master's thesis for which an extension was requested, the collection manager approved, and I actually made the change. It's unlikely to be edited in the future. I have locked this in H2 to prevent any further editing issues.
- druid:qx469wp3917: test object
- druid:zb733jx3137: test object
- druid:zx198bb7198: test object
